wrangler-core/src/main/java/io/cdap/directives/column/Keep.java

```diff
@@ -93,4 +103,24 @@ public Mutation lineage() {
     keep.forEach(column -> builder.relation(column, column));
     return builder.build();
   }
+  @Override
+  public Relation transform(RelationalTranformContext relationalTranformContext,
+                            Relation relation) {
+    Optional<ExpressionFactory<String>> expressionFactory = getExpressionFactory(relationalTranformContext);
+    if (!expressionFactory.isPresent()) {
+      return new InvalidRelation("Cannot find an Expression Factory");
+    }
+    Map<String, Expression> keepCol = generateColumnExpMap(keep.stream().collect(Collectors.toList()),
+            expressionFactory.get());
+    return relation.select(keepCol);
+  }
+
+  private Optional<ExpressionFactory<String>> getExpressionFactory(RelationalTranformContext ctx) {
```

Comment:
This function seems to be common across directives. can we move this to `Directive` interface or move it to a util class